docs/DafnyRef/Types.md

```diff
@@ -3654,6 +3654,31 @@ inductive datatype for trees may be updated as follows:
 node.(left := L, right := R)
 ```
 
+The operator `<` is defined for two operands of the same datataype.
+It means _is properly contained in_. For example, in the code
+```dafny
+datatype X = T(t: X) | I(i: int)
+method comp() {
+  var x: X := T(I(0));
+  var y: X := I(0);
+  var z: X := I(1);
```

Comment:
I suggest leaving off the `: X` to reduce clutter. With the definitions of `T` and `I` just above it, I don't think there's much chance of confusion.